### PR TITLE
Add converted LaTex files from Sui white paper to docs directory

### PR DESCRIPTION
**Pull request by Clay-Mysten**
For review only!

Let's get these in better shape before shipping.

### CODE REVIEW

**Comment by gdanezis** [comment]
Quick Q: should we verion the whitepaper? Say starting here at v0.1, and aiming for v1.0 to be mainnet?

Ideally we would extend the whitepaper with more details on testnet v0.2, pre-mainnet v0.3 and then mainnet to be an accurate record of what the system does when live.

File: doc/paper/markdown/outline.tex.md [hunk 1]
```diff
@@ -0,0 +1,1293 @@
+# Introduction
+
+is a decentralized permissionless smart contract platform biased towards
```
Comment (asonnino) [inline doc/paper/markdown/outline.tex.md:3]: What is the subject of this sentence?

File: doc/paper/markdown/outline.tex.md [hunk 2]
```diff
@@ -0,0 +1,1293 @@
...
+low-latency management of assets. It uses the Move programming language
+to define assets as objects that may be owned by an address. Move
+programs define operations on these typed objects including custom rules
+for their creation, the transfer of these assets to new owners, and
+operations that mutate assets.
+
+is maintained by a permissionless set of authorities that play a role
```
Comment (asonnino) [inline doc/paper/markdown/outline.tex.md:10]: Same. I suspect the LaTex paper uses a macro `\sysname` referring to 'Suis' that has not been converted correctly.

File: doc/paper/markdown/outline.tex.md [hunk 3]
```diff
@@ -0,0 +1,1293 @@
...
+reference="sec:move"} describing the programming model using the Move
+language, and Sect. [4](#sec:system){reference-type="ref"
+reference="sec:system"} describing the operations of the permissionless
+decentralized system that ensures safety, liveness and performance for .
+
+# smart contract programming
+
+[]{#sec:move label="sec:move"}
+
+smart contracts are written in the Move[@move_white] language. Move is
```
Comment (asonnino) [inline doc/paper/markdown/outline.tex.md:41]: Sui

File: doc/paper/markdown/outline.tex.md [hunk 4]
```diff
@@ -0,0 +1,1293 @@
...
+### Modules {#sec:modules}
+
+A Move program is organized as a set of modules, each consisting of a
+list of struct declarations and function declarations. A module can
+import struct types from other modules and invoke functions declared by
+other modules.
+
+Values declared in one Move module can flow into another--e.g., module
+in the example above could be defined in a different module than the
+module defining . This is different from most smart contract languages,
```
Comment (asonnino) [inline doc/paper/markdown/outline.tex.md:104]:
```suggestion
module defining. This is different from most smart contract languages,
```

**Review by asonnino — COMMENTED** [review]
I didn't finish a full pass yet but I see two recurring issues with the latex-MD converter:
- Latex macro do not always seem to be correctly rendered
- Latex ignores most of the spaces (eg. before "." and ",") and automatically fix them. It seems that MD does not.

File: doc/paper/markdown/outline.tex.md (hunk 2 above)
Comment (Clay-Mysten) [reply doc/paper/markdown/outline.tex.md:10]: Yes, this comes from the intro file:

```
\sysname is a decentralized permissionless smart contract platform biased towards low-latency management of assets.
%
```

Comment (Clay-Mysten) [reply doc/paper/markdown/outline.tex.md:10]: See the issue for (not-much) progress on this front:
https://github.com/MystenLabs/sui/issues/1368

File: doc/paper/markdown/outline.tex.md (hunk 4 above)
Comment (randall-Mysten) [reply doc/paper/markdown/outline.tex.md:104]: this looks like an incomplete sentence - could I do an edit pass on the source doc before converting to .md?

**Comment by Clay-Mysten** [comment]
Bumping this to avoid:
https://github.com/MystenLabs/sui/pull/3818

And see if Francois and team think it's worth working together to resolve the aforementioned macro conversion issue.

**Comment by Clay-Mysten** [comment]
Closing this for now as low priority

**Pull request closed without merging**